tools/k8s_versions/update_k8s_versions.go

```diff
@@ -0,0 +1,222 @@
+package main
```

Comment:
nit - I'm not sure about this but other go sources in the repo have a header. That said there is no check enforcing that....

Reply:
```suggestion
// Copyright Splunk Inc.
// SPDX-License-Identifier: Apache-2.0

package main
```

Reply:
makes sense!